engine/message/db.go

```diff
@@ -277,6 +279,22 @@ func NewDB(ctx context.Context, db *sql.DB, c *Config, a alertlog.Store) (*DB, e
 			) select distinct msg_id, alert_id, user_id, cm_id from disabled
 		`),
 
+		failSMSVoice: p.P(`
+			update outgoing_messages msg
+			set
+				last_status = 'failed',
+				last_status_at = now(),
+				status_details = 'twilio disabled',
```

Comment:
Maybe we should generalize this a bit?

```suggestion
				status_details = 'SMS/Voice support not enabled by administrator',
```